pkg/cliplugins/helpers/helpers.go

```diff
@@ -62,3 +65,20 @@ func IsValid(cluster logicalcluster.Name) bool {
 
 	return cluster.HasPrefix(v1alpha1.RootCluster) || cluster.HasPrefix(logicalcluster.New("system"))
 }
+
+func ClusterHasResource(gvk schema.GroupVersionKind ,discoveryClient discovery.DiscoveryInterface) (bool, error) {
+	klog.Info(fmt.Sprintf("looking for grouversion %v", gvk.GroupVersion()))
```

Comment:
Use `Infof` :)

Reply:
I want to remove all those log prints anyway. But indeed will for next time.

Reply:
for some reason the Cluster(currentClusterName) seem to always have the `Workspace` crd. 
I expected that this discovery client would belong to the current workspace. Am I wrong here?

Reply:
We install the `ClusterWorkspace` CRD in every workspace, IIRC

Reply:
`kubectl api-resources | grep Workspace`  will show nothing in a Universal workspace

Reply:
you are right there are clusterworkspaces